app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,25 @@
+class CommentsController < ApplicationController
+  def new
+    @user = User.find(params[:user_id])
+    @post = @user.posts.find(params[:post_id])
+    @comment = @post.comment_entries.build
+  end
+
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = @post.comment_entries.build(comment_params)
+    @comment.user = current_user
+
+    if @comment.save
+      redirect_to post_path(@post), notice: 'Comment was successfully created.'
+    else
+      render :new
+    end
+  end
```

Comment:
- [ ] I cannot create a comment either. Kindly fix that. I think this has to do with the `comment_params` or so.

  ![image](https://github.com/Bellagirl-maker/blog_app/assets/54848363/7bf968ed-a012-4b3d-ae6e-e937b01d448a)

Reply:
@DammyShittu done with this change